how-it-works/2_chainkeytech/04_chain_key_tokens.subpage.md

```diff
@@ -0,0 +1,66 @@
+
+This traditional off-chain approach of wrapping works well from a functional perspective, but has the major drawback of involving an intermediary whose integrity is crucial for security of the wrapping of the token. A main problem is that the intermediary can get compromised, e.g., be hacked, defrauded by an insider, or go out of business, which may result in a total loss of the underlying tokens in the worst case. Mitigations like multi-signature schemes with keys held by multiple parties to try to decentralize the intermediary can provide some mitigation, but do not change the fact that wrapping is done by an off-chain entity or group of entities. In short, this architecture is not fully decentralized. Overall, this traditional way of realizing wrapped tokens is not desirable for reasons of security, risk, and the inherent centralization it comes with.
+
+Another potential risk comes into play with those wrapping architectures: Ideally, a wrapped token is always 1:1 backed with the underlying token, but in practice, the intermediary can use the tokens held in custody to create profit, e.g., by using them for risky investments. In the worst case, this can, if things go wrong, lead to the loss of tokens and a depegging of the wrapped token.
+
+## Chain-Key Tokens
+
+Chain-key tokens, e.g., Chain-Key Bitcoin, are an advanced cryptography-based replacement for wrapped tokens offering stronger security: With chain-key tokens, all operations are performed completely on chain by smart contracts, not at all involving off-chain intermediaries. This eliminates the intermediary-related risks of traditional wrapping architectures. Replacing wrapping with on-chain operations only becomes possible through the native integrations between blockchain networks and particularly the use of advanced cryptography — *chain-key signatures*. Chain-Key Bitcoin, or ckBTC in short, is the first chain-key token available on ICP. It is a ‘twin’ of real bitcoin available on the ICP blockchain, backed 1:1 with real bitcoin that is held on chain on ICP by a canister smart contract.
+
+We next give an overview on how chain-key tokens function based on a native chain integration and advanced cryptography.
+
+### Architecture
+
+The architecture of any chain-key token on ICP is using the following building blocks as foundation:
+
+1. Some form of native integration with another blockchain that hosts the underlying token, e.g., the Bitcoin network and its only token, Bitcoin, must be available. This integration must allow canisters on ICP to query balances of addresses of the underlying token on its native chain as well as send transactions to the underlying chain. For underlying chains that use the UTXO accounting model like Bitcoin, the UTXOs of addresses must be queryable as well as those are required to assemble transactions. This integration with the native chain must be done such that no intermediaries are required, i.e., is completely decentralized and realized on chain.
```

Comment:
"For underlying chains that use the UTXO accounting model like Bitcoin, the UTXOs of addresses must be queryable as well as those are required to assemble transactions."

I think this sentence can be removed. I don't see the benefit of bringing up the UTXO model here.

Reply:
I would suggest to replace all "i.e." and "e.g.". It's too formal for a web page in my opinion.

Reply:
1st comment: done
2nd comment: for later